content/en/docs/tasks/inject-data-application/define-environment-variable-container.md

```diff
@@ -35,6 +35,11 @@ The `env` and `envFrom` fields have different effects.
 You can read more about [ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables)
 and [Secret](/docs/tasks/inject-data-application/distribute-credentials-secure/#configure-all-key-value-pairs-in-a-secret-as-container-environment-variables).
 
+The character range of the environment variable name can be seen in
+[Environment variables names](/docs/concept/configuration/environment/environment-variables-names)
+If the `RelaxedEnvironmentVariableValidation` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled,
+all [printable ASCII characters](https://www.ascii-code.com/characters/printable-characters) except "=" may be used for environment variable names.
```

Comment:
I (strongly) recommend putting this advice under its own heading, within the “Using environment variables inside of your config” heading. We should explain the default behavior (restricting what characters are allowed) and the experimental behavior that allows a wider range of characters.